libraries/api-documenter/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "api-documenter",
+  "version": "0.1.0",
+  "description": "Read JSON files from api-extractor, generate documentation pages",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/web-build-tools"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "gulp",
+    "clean": "gulp clean",
+    "test": "gulp test"
+  },
+  "devDependencies": {
+    "@microsoft/node-library-build": "~3.2.7",
+    "@types/colors": "1.1.1",
+    "@types/es6-collections": "0.5.29",
+    "@types/es6-promise": "0.0.32",
+    "@types/fs-extra": "0.0.37",
+    "@types/js-yaml": "~3.5.31",
+    "@types/node": "6.0.62",
+    "gulp": "~3.9.1"
+  },
+  "dependencies": {
+    "@microsoft/api-extractor": "2.3.2",
+    "colors": "~1.1.2",
+    "fs-extra": "~0.26.7",
+    "js-yaml": "~3.8.4"
+  }
+}
```

Comment:
I don't think so, since shouldPublish=false for api-documenter.

**test-web-library-build** does not have .npmignore for the same reason

---
In reply to: [134041055](https://github.com/Microsoft/web-build-tools/pull/325#discussion_r134041055) [](ancestors = 134041055)